setup.py

```diff
@@ -211,4 +206,4 @@
     url='https://github.com/canonical-ols/talisker',
     version='0.21.3',
     zip_safe=False,
-)
+)
```

Comment:
This seems to be an unwanted change. You probably deleted at the empty line at the end of the file.